controllers/deviceprofile_syncer.go

```diff
@@ -9,12 +9,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	ctrlmgr "sigs.k8s.io/controller-runtime/pkg/manager"
 
-	coremetacli "github.com/charleszheng44/device-controller/clients/core-metadata"
 	"github.com/edgexfoundry/go-mod-core-contracts/models"
 	"github.com/go-logr/logr"
+	coremetacli "github.com/openyurtio/device-controller/clients/core-metadata"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
```

Comment:
please add license header. all others files should be check.

Reply:
Thanks for reminding, the license headers are all added.